themes/default/theme/stencil/src/store/actions/user.ts

```diff
@@ -0,0 +1,13 @@
+import { TypeKeys } from "./index";
+
+export interface GetUser {
+    type: TypeKeys.GET_USER_INFO;
+}
+
+export const getUser = () => (dispatch, _getState) => {
+    console.log("getUser");
```

Comment:
im guessing this was this in there just for testing?

Reply:
Ah good catch!